extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -3683,7 +3733,7 @@ var globalRequire = require;
 				 */
 				this.pauseAd = function(ad)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "ad". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)